snapcraft/internal/deprecations.py

```diff
@@ -28,6 +28,7 @@
     'dn1': "The 'snap' keyword has been replaced by 'prime'.",
     'dn2': "Custom plugins should now be placed in 'snap/plugins'.",
     'dn3': "Assets in 'setup/gui' should now be placed in 'snap/gui'.",
+    'history': "The 'history' keyword has been replaced by 'revisions'.",
```

Comment:
This needs to use 'dn4' for the ID (that's mostly historical at this point, but it conforms with the rest). Also, s/keyword/command/ . Finally, was it replaced by `revisions` or `list-revisions` (since one is an alias of the other...)?